react-native-libraries.json

```
@@ -15971,5 +15971,17 @@
     "ios": true,
     "android": true,
     "newArchitecture": true
-  }
+  },
+  {
+  "githubUrl": "https://github.com/superwall/expo-superwall",
+  "npmPkg": "https://www.npmjs.com/package/expo-superwall",
+  "examples": [
+    "https://github.com/superwall/expo-superwall/tree/main/example",
+  ],
+  "images": ["https://user-images.githubusercontent.com/3296904/158817914-144c66d0-572d-43a4-9d47-d7d0b711c6d7.png"],
+  "ios": true,
+  "android": true,
+  "windows": false,
+  "newArchitecture": true
+}
```

Hey @Makisuo, thanks for adding a new package to the directory! 👍 

Few suggestions below:
* `examples` array contains trailing comma, which causes syntax error
* `npmPkg` should be filled only with package name, but it's redundant in this case, see:
  * https://github.com/react-native-community/directory?tab=readme-ov-file#npmpkg
* `images` array should not include marketing resources, like logos, see:
  * https://github.com/react-native-community/directory?tab=readme-ov-file#images
* platform `false` flags can be safely omitted

```suggestion
  {
    "githubUrl": "https://github.com/superwall/expo-superwall",
    "examples": [
      "https://github.com/superwall/expo-superwall/tree/main/example"
    ],
    "ios": true,
    "android": true,
    "newArchitecture": true
  }
```

With this changes in, the CI should be passing, and entry should be ready to merge.